content/kotlin/concepts/math-methods/terms/tanh/tanh.md

```diff
@@ -0,0 +1,43 @@
+---
+Title: '.tanh()'
+Description: 'Computes the hyperbolic tangent of an angle.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Functions'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **`.tanh()`** method in Kotlin's `math` class computes the hyperbolic tangent of an angle given in radians.
+
+## Syntax
+
+```pseudo
+tanh(x)
+```
+
+The function accepts one mandatory parameter `x`. The argument can be of `Double` or `Float` datatype and it returns a value of the same type.
```

Comment:
Since the first sentence here is moved to the paragraph before syntax, remove it. And then turn the remaining sentence into a point using the variable `x` as shown in the example below:

`` -`y`: Stores the computed hyperbolic tangent value ``

Reply:
I am not getting this properly but still I have made the changes. Could you please confirm ?